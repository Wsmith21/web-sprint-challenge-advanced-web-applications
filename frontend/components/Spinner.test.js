import React from 'react';
import { render } from '@testing-library/react';
import Spinner from './Spinner'; // Replace './Spinner' with the correct path to your Spinner component

test('renders a default spinner', () => {
  const { container } = render(<Spinner />);
  // Add assertions to test the default rendering of the spinner
  expect(container).toMatchSnapshot(); // Use snapshot testing or other specific assertions
});