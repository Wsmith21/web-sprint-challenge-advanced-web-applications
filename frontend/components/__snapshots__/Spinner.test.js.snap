// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a default spinner 1`] = `<div />`;
